{
  "comments": [
    {
      "key": {
        "uuid": "b7dba2f9_ba2ceb54",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 5,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-06-11T10:57:48Z",
      "side": 1,
      "message": "please test if \u0027git\u0027 step is safe to retry in case it gets interrupted in middle (e.g. connection cut/shutdown the git server) ... in regards to it not/leaving directory which would prevent it trying again",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "5f6eaab1_a1fb8aa0",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 5,
      "author": {
        "id": 1017759
      },
      "writtenOn": "2020-06-11T11:39:29Z",
      "side": 1,
      "message": "Checked locally - ansible Git module seems to delete partial git folder and creates a new one on the next try",
      "parentUuid": "b7dba2f9_ba2ceb54",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "a82bf556_6b03e84c",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 9,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-06-11T10:57:48Z",
      "side": 1,
      "message": "TYPO: reTries",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "6dcc2fb9_35b53989",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 9,
      "author": {
        "id": 1017759
      },
      "writtenOn": "2020-06-11T11:42:17Z",
      "side": 1,
      "message": "Ack",
      "parentUuid": "a82bf556_6b03e84c",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "57f7d0b8_08e28392",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 10,
      "author": {
        "id": 1005532
      },
      "writtenOn": "2020-06-11T10:58:10Z",
      "side": 1,
      "message": "If the repo is busy I suggest to increase the delay and retries (10 retries and 15 seconds delay? )",
      "range": {
        "startLine": 10,
        "startChar": 9,
        "endLine": 10,
        "endChar": 10
      },
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "9238555e_5d9096ee",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 10,
      "author": {
        "id": 1017759
      },
      "writtenOn": "2020-06-11T11:42:17Z",
      "side": 1,
      "message": "I think it will be overkill, but if you insist I can change it to 5 retries with 15 seconds delay. More will be too much in my oppinion",
      "parentUuid": "57f7d0b8_08e28392",
      "range": {
        "startLine": 10,
        "startChar": 9,
        "endLine": 10,
        "endChar": 10
      },
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "9ab07f56_87d0fcb8",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 12,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-06-11T11:06:44Z",
      "side": 1,
      "message": "i would suggest to use the preferred method of testing for success/failure of registered variables\n\n  until: tempest_conf_repo is not failed\n  # or\n  until: tempest_conf_repo is succeeded\n\nas per e.g.:\n\n- https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#the-when-statement (bottom part of the section)\n- https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#test-syntax\n- or https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.5.html#jinja-tests-used-as-filters\n\ni know you are not using the filters here so that is ok, just that there is explicit test for success/failure and so no need to compare to boolean",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "542fade1_4d537ac4",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 15,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-06-11T10:57:48Z",
      "side": 1,
      "message": "i\u0027m not sure, but i would guess this is not needed,\nas i would expect that \u0027git: retries:3 untile:not-failed\u0027\nwill cause failure after 3 retries running out without success",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "33b131b6_abf9b442",
        "filename": "plugins/tempest/tasks/configure/method/git.yml",
        "patchSetId": 1
      },
      "lineNbr": 15,
      "author": {
        "id": 1017759
      },
      "writtenOn": "2020-06-11T11:42:17Z",
      "side": 1,
      "message": "There is an ignore_errors in patch 1 so  without this fail it will always pass.",
      "parentUuid": "542fade1_4d537ac4",
      "revId": "6f9e14214be074ca0c4cd8711357d20678bd300a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}